misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -157,8 +138,28 @@ class HibernateModule(
           )
         }).asSingleton()
     install(ServiceModule<SchemaValidatorService>(qualifier)
-        .dependsOn<SchemaMigratorService>(qualifier)
-        .dependsOn<SessionFactoryService>(qualifier))
+        .dependsOn<SchemaMigratorService>(qualifier))
+
+    // Bind SessionFactoryService as implementation of TransacterService.
```

Comment:
Note that I moved this block (verbatim) from above to keep the bindings in order of usage. Since `SchemaValidatorService` and `SchemaMigratorService` no longer depend on `SessionFactoryService`, this has been moved to ensure this configuration block continues to read as linearly as possible.

Reply:
FYI @tirsen this might collide with you?